connect/src/com/telenor/connect/id/Claims.java

```diff
@@ -8,6 +8,7 @@
 
     public static final String OPENID = "openid";
     public static final String PROFILE = "profile";
```

Comment:
I don't understand this. "openid" is not a claim and we do not support the "profile" claim yet (only the profile scope which "maps" to several different claims, including the profile and name claims). I think I need to take a closer look at this code.